Gems/ROS2/Code/Source/RobotImporter/Pages/PrefabMakerPage.cpp

```diff
@@ -54,5 +56,10 @@ namespace ROS2
     {
         return m_success;
     }
+    bool PrefabMakerPage::isUseArticulations() const
```

Comment:
Naming of member methods: names should start capitalized. Perhaps remove the "is", alternatively "IsUseArticulationsChecked"